src/parsers/srdf.hxx

```diff
@@ -200,12 +200,35 @@ namespace pinocchio
       assert(false && "no rotor params found in the srdf file");  
       return false; // warning : uninitialized vector is returned
     }
-    
+
     template<typename Scalar, int Options, template<typename,int> class JointCollectionTpl>
     typename ModelTpl<Scalar,Options,JointCollectionTpl>::ConfigVectorType
     getNeutralConfiguration(ModelTpl<Scalar,Options,JointCollectionTpl> & model,
                             const std::string & filename,
                             const bool verbose) throw (std::invalid_argument)
+    {
```

Comment:
This function is now deprecated. It would be better to not modify its content.

Reply:
Since the function is still valid, the change is meant to keep its current behaviour unmodified in order to not affect any packages downstream.

I am not adding any changing the behaviour of the function here. The function remains the same for the users.

Reply:
Yes, but not the content. This also means that if you've introduced a bug here, then the bug will appear to the user too.

Reply:
Would you prefer that I duplicate the code?

Reply:
I would prefer to not modify the code of getNeutralConfiguration, some having some duplicate codes. It is just a safer procedure when dealing with deprecated functions.

Reply:
Done.